config/network.conf.json

```diff
@@ -2,55 +2,29 @@
   "explorerUrl": "http://explorer.symboldev.network",
   "faucetUrl": "http://faucet-01.symboldev.network",
   "defaultNetworkType": 152,
-  "defaultNode": {
-    "protocol": "http",
-    "hostname": "api-01.us-west-1.symboldev.network",
-    "port": 3000,
-    "url": "http://api-01.us-west-1.symboldev.network:3000"
+  "defaultNodeUrl": "http://api-01.ap-northeast-1.symboldev.network:3000",
+  "networkConfigurationDefaults": {
+    "maxMosaicDivisibility": 6,
+    "namespaceGracePeriodDuration": 2592000,
+    "lockedFundsPerAggregate": "10000000",
```

Comment:
is it wanted to store this number as a string here?

Reply:
The number would be used as Uint64 when creating a LockFundsTransaction. If the property is a number it may overflow.